docs/release/release_0_5_0.md

```diff
@@ -0,0 +1,911 @@
+
+## API Changes
+
+- Add .to_rgb_dict() to Theme class. ([napari/napari#4759](https://github.com/napari/napari/pull/4759))
+- Thick Slices (Dims  as nD-box instead of nD-point) ([napari/napari#5522](https://github.com/napari/napari/pull/5522))
+- Lock dimensions / axes while rolling ([napari/napari#5986](https://github.com/napari/napari/pull/5986))
+- Remove deprecated code from Labels layer ([napari/napari#6641](https://github.com/napari/napari/pull/6641))
+
+## Deprecations
+
+- MAINT: Remove unused function. ([napari/napari#5041](https://github.com/napari/napari/pull/5041))
+- MAINT: remove deprecated Alias. ([napari/napari#5043](https://github.com/napari/napari/pull/5043))
+- Fix typing in _app_model ([napari/napari#6059](https://github.com/napari/napari/pull/6059))
+- Rename `edge_*` attributes and references to `border_*` on points layer ([napari/napari#6402](https://github.com/napari/napari/pull/6402))
+- Add deprecation message to CallDefault function partly resolves issue#6257 ([napari/napari#6901](https://github.com/napari/napari/pull/6901))
+
+## Build Tools
+
+- ci(dependabot): bump WebFreak001/deploy-nightly from 1.1.0 to 1.2.0 ([napari/napari#5159](https://github.com/napari/napari/pull/5159))
+- ci(dependabot): bump convictional/trigger-workflow-and-wait from 1.6.1 to 1.6.4 ([napari/napari#5293](https://github.com/napari/napari/pull/5293))
+- ci(dependabot): bump bruceadams/get-release from 1.2.3 to 1.3.2 ([napari/napari#5294](https://github.com/napari/napari/pull/5294))
+- ci(dependabot): bump convictional/trigger-workflow-and-wait from 1.6.4 to 1.6.5 ([napari/napari#5384](https://github.com/napari/napari/pull/5384))
+- ci(dependabot): bump WebFreak001/deploy-nightly from 1.2.0 to 2.0.0 ([napari/napari#5385](https://github.com/napari/napari/pull/5385))
+- ci(dependabot): bump docker/build-push-action from 3 to 4 ([napari/napari#5523](https://github.com/napari/napari/pull/5523))
+- ci(dependabot): bump toshimaru/auto-author-assign from 1.6.1 to 1.6.2 ([napari/napari#5524](https://github.com/napari/napari/pull/5524))
+- fix various keybinding/shortcut bugs ([napari/napari#5604](https://github.com/napari/napari/pull/5604))
+- ci(dependabot): bump peter-evans/create-pull-request from 4 to 5 ([napari/napari#5792](https://github.com/napari/napari/pull/5792))
+- ci(dependabot): bump actions/setup-python from 2 to 4 ([napari/napari#6201](https://github.com/napari/napari/pull/6201))
+- ci(dependabot): bump docker/login-action from 2.1.0 to 3.0.0 ([napari/napari#6263](https://github.com/napari/napari/pull/6263))
+- ci(dependabot): bump actions/checkout from 2 to 4 ([napari/napari#6264](https://github.com/napari/napari/pull/6264))
+- ci(dependabot): bump docker/build-push-action from 4 to 5 ([napari/napari#6280](https://github.com/napari/napari/pull/6280))
+- ci(dependabot): bump docker/metadata-action from 4 to 5 ([napari/napari#6282](https://github.com/napari/napari/pull/6282))
+
+## Documentation
```

Comment:
Is it worth breaking out the `maintenance` label docs PRs to a different section?
maybe have a section:
`Copy improvements`
&
`Maintenance`
I can go and do some tagging if needed.